qiskit_experiments/library/quantum_volume/qv_experiment.py

```diff
@@ -74,29 +73,29 @@ def __init__(
         self,
         qubits: Sequence[int],
         backend: Optional[Backend] = None,
-        trials: Optional[int] = 100,
-        seed: Optional[Union[int, SeedSequence, BitGenerator, Generator]] = None,
         simulation_backend: Optional[Backend] = None,
+        **experiment_options,
     ):
         """Initialize a quantum volume experiment.
 
         Args:
             qubits: list of physical qubits for the experiment.
             backend: Optional, the backend to run the experiment on.
-            trials: The number of trials to run the quantum volume circuit.
-            seed: Optional, seed used to initialize ``numpy.random.default_rng``
-                  when generating circuits. The ``default_rng`` will be initialized
-                  with this seed value everytime :meth:`circuits` is called.
             simulation_backend: The simulator backend to use to generate
                 the expected results. the simulator must have a 'save_probabilities'
                 method. If None :class:`AerSimulator` simulator will be used
                 (in case :class:`AerSimulator` is not
                 installed :class:`qiskit.quantum_info.Statevector` will be used).
-        """
-        super().__init__(qubits, analysis=QuantumVolumeAnalysis(), backend=backend)
+            experiment_options: A catch-all for any experiment options. See
+                :py:meth:`_default_experiment_options` for valid parameters.
```

Comment:
```suggestion
                :py:attr:`experiment_options` for valid parameters.
```
Since this is API doc, referring to protected method is bit weird, i.e. asking user to read the code.

Reply:
Agreed docs should reference where in the API docs the options are listed

Reply:
I completely agree, and in hindsight should have looked at how `experiment_options` was referenced in existing docstrings. Addressed in bbe6821.